packages/cli/src/splitRunner/splitRunner.ts

```diff
@@ -1,231 +1,44 @@
-import { Group } from "../dependencyManager/types";
-import { removeIndexesFromSourceCode } from "../helper/file";
+import path from "path";
+import { Worker } from "worker_threads";
 import DependencyTreeManager from "../dependencyManager/dependencyManager";
+import { Group } from "../dependencyManager/types";
 import { File } from "./types";
-import Parser from "tree-sitter";
-import assert from "assert";
-import { getLanguagePlugin } from "../languagesPlugins";
-import { DepExport } from "../languagesPlugins/types";
 
 class SplitRunner {
```

Comment:
Let's delete this class, become kind of useless now. We can do a simple method to replace it.
Maybe in another PR